plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -336,6 +336,18 @@ func ClusterRoles() []rbac.ClusterRole {
 			},
 		},
 		{
+			// a role to use for both out-of-tree or in-tree cloud-controller-manager
```

Comment:
Where are these values coming from? Do you have some docs you can link to indicate what a cloud-controller-manager would need to manage or where we would look for things if they updated?

Reply:
@wlan0 will be posting something for this soon :)

Reply:
https://github.com/kubernetes/kubernetes.github.io/pull/5928